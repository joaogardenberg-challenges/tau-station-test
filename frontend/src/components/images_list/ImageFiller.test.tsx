import { render, screen } from '@testing-library/react'
import ThemeProvider from 'providers/Theme'

const ImageFiller = require('./ImageFiller').default

const renderWithTheme = (ui: any, options: any = {}) =>
  render(ui, { wrapper: ThemeProvider, ...options })

describe('Image Filler', () => {
  beforeEach(() => {})

  it('renders successfully', () => {
    const { container } = renderWithTheme(<ImageFiller />)
    expect(container).toMatchSnapshot()
  })
})
