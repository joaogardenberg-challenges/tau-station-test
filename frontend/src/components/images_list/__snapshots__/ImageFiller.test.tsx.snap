// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Filler renders successfully 1`] = `
<div>
  <li
    class="sc-bdfBQB gcgrmU"
  />
</div>
`;
